code/modules/procedural mapping/planetGenerator/story_generator/_story_generator.dm

```diff
@@ -0,0 +1,658 @@
+	return entries
+
+/datum/story_theme/proc/get_fauna_entries()
+	var/list/entries = list()
+	switch(planet_style)
+		if("beach planet")
+			entries += "Observed several species of crabs along the shoreline. Mostly harmless, but territorial."
+			entries += "The local wildlife includes some surprisingly intelligent capybaras. They seem curious about our presence."
+			entries += "Encountered aggressive frogs near the wetlands. Their jumping reach is remarkable."
+		if("desert planet")
+			entries += "The desert lizards here are abundant. Most scatter when approached, but some hold their ground."
+			entries += "Massive goliath-class creatures spotted in the distance. Avoiding direct contact."
+			entries += "Strange insectoid life forms emerge at dusk. Classification pending."
+		if("frozen planet")
+			entries += "Pack of wolves spotted near the perimeter. They're watching us. Recommend staying inside after dark."
+			entries += "Polar bears are active in this region. One investigated the camp last night. Security protocols updated."
+			entries += "Encountered what the team is calling 'wendigos' - bipedal predators adapted to the cold. Extremely dangerous."
+		if("grass planet")
+			entries += "The grasslands support diverse herbivores - everything from cattle-like creatures to deer."
+			entries += "Discovered aggressive cockatrice specimens. Their behavior suggests territorial nesting."
+			entries += "Local fauna includes various domesticated species gone feral. The ecology here is fascinating."
+		if("jungle planet")
+			entries += "The jungle teems with life. Parrots, monkeys, and stranger things in the canopy."
+			entries += "Poison dart frogs are common in the undergrowth. Specimens collected for study, with extreme caution."
+			entries += "Spotted a large predatory cat - possibly a panther variant. It's been tracking us for two days."
+		if("lava planet")
+			entries += "Goliath-class megafauna dominate this hellscape. Their biology shouldn't work, but here they are."
+			entries += "Basilisk creatures can somehow survive the heat. They burrow through rock like water."
+			entries += "The local 'hivelords' are territorial and aggressive. Lost a drone to one yesterday."
+		if("wasteland planet")
+			entries += "The ruins are infested with roaches. Not the Earth variety - these are adapted to radiation."
+			entries += "Reanimated corpses wander the wastes. Necromantic phenomenon or biological? Unclear."
+			entries += "Whatever caused the apocalypse here, the survivors have mutated into something barely recognizable."
+		if("unknown planet")
+			entries += "The native lifeforms defy classification. Grey humanoids with technology beyond our understanding."
+			entries += "Encountered autonomous drones of alien design. They ignore us unless we approach designated zones."
+			entries += "Strange polyp-like organisms dot the landscape. They react to movement with surprising hostility."
```

Comment:
Not sure this fits considering the established "lore" of the ayylmaos, the mothership, them being a clone species living almost exclusively in fuckhuge ships, etc.
Most of that lore was established by optimism/b4thunderiz in the mothership vault and ZAM snacks.
Could work if it's an ANCIENT ruin, i'm talking several hundreds of years old (current spess is around Y2570 or so, so it'd work if it was 2080 or some shit).